pallets/gas/src/tests.rs

```diff
@@ -46,6 +46,41 @@ fn simple_value_tree() {
     });
 }
 
+#[test]
+fn value_tree_with_all_kinds_of_nodes() {
```

Comment:
what about making it more complicated by cutting value from unspecified and specified nodes as well?

Reply:
mb I can provide a new test `can_cut_reserved_nodes` for this?

Reply:
> mb I can provide a new test `can_cut_reserved_nodes` for this?

> Hope you are talking about only about testing, cause we didn't expect atm that cut node can be gas less

no-no
I mean cutting not only root, but specified and unspecified nodes as well and checking node's balances and such